styles/contactUs.css

```diff
@@ -0,0 +1,216 @@
+* {
+  box-sizing: border-box;
+}
+section.contactUs {
+  width: 100%;
+}
+section.contactUs > h2 {
+  font-family: "Ubuntu", sans-serif;
+  font-size: 32px;
+  letter-spacing: 0.02em;
+  color: #222222;
+  font-weight: 700;
+  line-height: 37px;
+  padding-left: 13vw;
+  text-align: left;
+}
+.contactUs__container {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: #fafaff;
+  width: 80vw;
+  height: 40vw;
+  margin: 0 auto;
+  box-shadow: 6px 6px 20px 0 rgba(255, 255, 255, 0.5);
+}
+.contactUs__footer {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  flex-basis: 40%;
+  height: 100%;
+  background-color: #ff9a3d;
+  padding: 3% 10% 3% 4%;
+  box-shadow: 6px 6px 20px 0 rgba(255, 255, 255, 0.5);
+}
+.contactUs__footer > img {
+  width: 4vw;
+  height: auto;
+  transform: rotate(-30deg);
+}
+.contactUs__paragraph {
+  color: #fff;
+  font-size: 18px;
+}
+.contactUs__paragraph a {
+  display: block;
+  color: #fff;
+}
+.contactUs__paragraph:nth-of-type(1) {
+  font-family: "Raleway", sans-serif;
+  line-height: 31px;
+  flex-grow: 1;
+  letter-spacing: 0.04em;
+}
+.contactUs__paragraph:nth-of-type(2) {
+  font-family: "Ubuntu", sans-serif;
+  line-height: 25px;
+  letter-spacing: 0.02em;
+}
+.contactUs__paragraph:nth-of-type(2) span {
+  font-weight: 700;
+}
```

Comment:
Sprawdź czy jak nie przekązemy tego font-weight do tego stylu wyżej nie bedzie działało podobnie, wtedy nie bedziemy muslie tworzyc ".contactUs__paragraph:nth-of-type(2) span"

Reply:
nie zadziała....., ale zadziała jak span wymienimy na strong, nie jestem do końca pewien czy to semantycznie poprawne, ale tak póki co zrobiłem